backend/common/utils/dataclass.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import asdict, is_dataclass
+
+
+def convert_dataclass_to_dict(data, remove_nones=True):
```

Comment:
i think for this i would prefer if we either don't set a default or default it to `False`. if you consider what a generic "convert dataclass to dict" util would do, it feels more intuitive to not alter the data at all

Reply:
Nevermind, I just did a little refactoring and added a cellguide-specific wrapper for the convert dataclass to dict.